examples/pipelines/aws/buildspec.yaml

```diff
@@ -14,32 +14,18 @@ phases:
       python: 3.8
   pre_build:
     commands:
-      ## Option 1 - Install the NeoLoad CLI python tool from Pypi
-      ## without samples, and from standard distro, use below
-      #- python3 -m pip install neoload
-
-      ## Option 2 - Install a custom version of the NeoLoad CLI python tool from source on Github
-      # get samples and specific branch of CLI for sla fastfail functionality
-      - git clone https://github.com/Neotys-Labs/neoload-cli.git && cd neoload-cli && git checkout topic-fastfail-command && git pull
-      # install the NeoLoad CLI Python client from above source pull
-      - python3 -m pip install -q .
-      # when installing from git source
-      - neoload_bin=$(find $(pyenv root) -type f -perm -100 -name neoload)
-      - export PATH=$(pyenv root)/shims:${neoload_bin/neoload/}:$PATH
-
+      - python3 -m pip install -q neoload
```

Comment:
We still need to clone  the repo to have sample projects. Add these 2 lines  :
\# get samples from Github
      - git clone https://github.com/Neotys-Labs/neoload-cli.git .

Reply:
Resolved in Commit fe8ff1a